AudioDeviceExample/README.md

```diff
@@ -26,14 +31,22 @@ Once you have configured your access token, build and run the example. You will
 
 <kbd><img width="400px" src="../images/quickstart/audio-device-launched.jpg"/></kbd>
 
+Tap the audio device button to select an audio device: 
+
+<kbd><img width="400px" src="../images/quickstart/select-audio-device.jpg"/></kbd>
+
+Once the audio device of your choice is selected, enter the room name.
+
+<kbd><img width="400px" src="../images/quickstart/enter-room-name.jpg"/></kbd>
+
 Tap the "Connect" button to join a Room. Once you've joined you will be sharing video but not audio. In order to playback audio from a remote Participant you will need a Client which supports audio recording. The easiest way to do this is to build and run the normal QuickStart [example](https://github.com/twilio/video-quickstart-swift/tree/2.0.0-preview/VideoQuickStart) and join the same Room.
```

Comment:
This portion should be updated since you can now run two instances of this example app just fine as long as one device is on iOS 11 and can publish audio.

Reply:
```Tap the "Connect" button to join a Room. Once you've joined you will be sharing video and audio if `ExampleAVEngineAudioDevice` is used. However the audio will not be shared if `ExampleCoreAudioDevice` is used. In order to playback audio from a remote Participant you will need a Client which supports audio recording. The easiest way to do this is to build and run the normal QuickStart [example](https://github.com/twilio/video-quickstart-swift/tree/2.0.0-preview/VideoQuickStart) and join the same Room.

After the remote Participant has joined you should be able to hear their audio. If you are using the `ExampleCoreAudioDevice`, watch out if both devices are in the same physical space, because `ExampleCoreAudioDevice` does not use echo cancellation.```